docs/points/points.rst

```diff
@@ -3,4 +3,99 @@
 Points
 ######
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Points provide a way for properly weighing Contacts. These Points have both triggers and actions. Each term is properly defined and a thorough understanding of how Points function ensures that your overall marketing automation process is successful.
+
+Point actions
+*************
+
+Point actions are those times when a Contact receives a change in their point total. These actions can be either positive or negative point changes and are based on a particular action as you determine.
+
+To add a new action:
+
+1. Click **Points > Point Actions > + New** (top right corner).
+
+.. image:: images/points-action.png
+    :alt: Screenshot of Points action
+
+2. In the main panel, there are four boxes for key information. Enter the appropriate information.
+
+   * **Name** - The name of your action. This is how the action displays in your list of actions, so choose an identifiable name.
+
+   * **Description** - Add a description to help you find certain actions. There may be more actions which are similar or more in-depth.
+
+   * **Change Points (+/-)** - The value change to set for the action. The ``+`` isn't necessary when adding Points. When subtracting Points, add the ``-`` symbol.
+
+   * **When a Contact…** - This is the behavior or action the Contact must complete to receive the Points.
```

Comment:
```suggestion
   * **When a Contact…** - This is the behavior or action the Contact must complete for the action to be triggered.
```

Reply:
We don't want to assume that points are being added, as they can equally be subtracted.

Reply:
Got it